test/test_nn.py

```diff
@@ -3539,6 +3591,30 @@ def _test_batchnorm_eval(self, test_type=torch.FloatTensor):
         self.assertEqual(res1, res2)
         self.assertEqual(grad1, grad2)
 
+        # track_running_stats=False
+        module = nn.BatchNorm1d(3, track_running_stats=False).type(test_type)
```

Comment:
Do you test track_running_stats=False for BatchNorm{2d,3d}?

Reply:
After discussion offline, we notice that this method originally only checks for 1d case. Considering that gradchecks for this case is done below, and that all three use the same underlying implementation, this should be fine.